lib/Monitor/RunFolder/Staging.pm

```diff
@@ -217,6 +220,12 @@ sub move_to_analysis {
 
     move( $self->runfolder_path(), $destination );
 
+    my $group = get_config()->{group};
```

Comment:
This needs a more descriptive config: use "analysis_group" within the already used "staging_areas", or something better....